crowdsec-docs/docs/appsec/installation.md

```diff
@@ -0,0 +1,130 @@
+
+# Pre-requisites
+
+To have a functional aplication security component, you need:
+ - Crowdsec security engine >= 1.5.6
+ - One of the compatible bouncers:
+
+
+<!-- @kka min nginx version-->
+| Name | Minimum Version |
+| --- | --- |
+| nginx | X.Y.Z |
+
+# Introduction
+
+Before jumping into the action, let's review the type of configuration items that are involved into the appsec component configuration:
+
+ - **acquisition configuration** tells on which interface and port is the service exposed, as well as the associated AppSec component configuration it will use.
+ - **appsec component configuration** tells which rules are loaded in inband (blocking) and out-of-band (non-blocking) 
+phases, [and allows to tweak the behavior of the component via the powerfull expr bindings](/appsec/rules.md). <!--@sbl we need anchor for the on_whatever and expr helpers -->
+
+ - **rules** allow you to write a [signature to detect and/or block malevolent requests](/appsec/rules.md).
+
+With that covered, let's jump into the installation!
+
+# Configuration: Collection install
+
+As often in crowdsec, the relevant pieces of configuration can be acquired by installing a collection, here we are going to use a collection inspired from the [CISA's catalog of known exploited vulnerabilities](https://www.cisa.gov/known-exploited-vulnerabilities-catalog). The goal is to target vulnerabilities that are popular and might be exploited by bad guys trying to break into your server:
+
+<!-- @tko fix collection name -->
+
+```
+cscli collections install crowdsecurity-waap/cisa-virtual-patching
+```
+
+Installing this collection brings the relevant appsec-rules (attack signatures), and the `crowdsecurity/vpatch` appsec-configuration (configuration for the appsec component).
+
+# Configuration: Application Security Component
+
+The appsec component is available as a data-source in crowdsec, and allows us to expose a port that the remediation engine / bouncer will send requests to, and gets verdict from. To enable it, we need to add the data source. This can be done by editing `/etc/crowdsec/acquis.yaml` or adding a new data source file in `/etc/crowdsec/acquis.d/`. We are going to add an aplication security component focused on detecting and blocking the exploitation of well-known vulnerabilities (virtual patching):
+
+```bash
+mkdir  -p /etc/crowdsec/acquis.d
+cat > /etc/crowdsec/acquis.d/appsec.yaml << EOF
+listen_addr: 127.0.0.1:4242
+appsec_config: crowdsecurity/vpatch
+source: appsec
+labels:
+  type: appsec
+EOF
+```
+
+The important lines are `listen_addr` that indicates on which interface/port the service listens too, and `appsec_config` that tells which configuration the appsec component will run.
```

Comment:
we should have this mentioned before the example code.

Also, it's not very clear to me what you mean by "`appsec_config` that tells which configuration the appsec component will run." ..

What is crowdsecurity/vpatch is it the type of behaviour ? is it targeting a fongi path ? config file ?